src/Resources/contao/dca/tl_leaflet_layer.php

```diff
@@ -14,6 +15,9 @@
 
 use MetaModels\CoreBundle\Contao\Hooks\ContentElementCallback;
```

Comment:
The `ContentElementCallback` could be removed and all calls can be replaced with the `LeafletLayerCallback` because they have the same has the same parent class.

Reply:
fixed with 65a8871f25cdd2ab82c1420dccda84309b166147